filebeat/module/elasticsearch/audit/test/test.log-expected.json

```diff
@@ -1,114 +1,242 @@
 [
     {
-        "@timestamp": "2018-06-19T05:16:15,549", 
-        "elasticsearch.audit.event_type": "authentication_failed", 
-        "elasticsearch.audit.layer": "rest", 
-        "elasticsearch.audit.origin_address": "147.107.128.77", 
-        "elasticsearch.audit.principal": "i030648", 
-        "elasticsearch.audit.uri": "/_xpack/security/_authenticate", 
-        "fileset.module": "elasticsearch", 
-        "fileset.name": "audit", 
-        "input.type": "log", 
-        "message": "[2018-06-19T05:16:15,549] [rest] [authentication_failed]        origin_address=[147.107.128.77], principal=[i030648], uri=[/_xpack/security/_authenticate]", 
-        "offset": 0, 
-        "prospector.type": "log", 
+        "@timestamp": "2018-06-19T05:16:15,549",
```

Comment:
I' wondering why the spacing at the end changed as these files are auto generated? Perhaps you opened them in your editor and it did some auto fixing? Or someone else did that in the past and the new one is the generated one? Could also have been a script change.

Reply:
It's definitely something my editor auto-fixed, not something I did manually. In any case I'm going to be splitting up the log files for the two formats now (per previous comment) and so I'll split up the corresponding expected files too, so this "cleanup" should go away.